test/models/role_test.rb

```diff
@@ -1,14 +1,7 @@
-# frozen_string_literal: true
-
 require "test_helper"
 
 class RoleTest < ActiveSupport::TestCase
-  context "associations" do
-    should have_and_belong_to_many(:staff_accounts).join_table(:staff_accounts_roles)
-    should belong_to(:resource).optional
-  end
-
-  context "validations" do
-    should validate_inclusion_of(:resource_type).in_array(Rolify.resource_types).allow_nil
-  end
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
Do we need tests here?

Reply:
I don't think we need to test the roles. They are being implemented by rolify, and they're mostly just data.

However, now that you mention this, I think we should write tests for `authorizable`'s public methods.

Reply:
Agreed^